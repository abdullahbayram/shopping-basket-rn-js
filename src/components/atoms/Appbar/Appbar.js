import { Appbar as PaperAppbar } from 'react-native-paper';

export default PaperAppbar;
