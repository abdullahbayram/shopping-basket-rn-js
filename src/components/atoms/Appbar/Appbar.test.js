import React from 'react';
import { render } from '@testing-library/react-native';
import Appbar from '.';

test('Appbar renders correctly', () => {
  const view = render(<Appbar />);
  expect(view.toJSON()).toMatchSnapshot();
});
