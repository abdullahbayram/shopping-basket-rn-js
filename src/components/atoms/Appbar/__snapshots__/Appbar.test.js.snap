// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Appbar renders correctly 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "rgba(255, 251, 254, 1)",
      "height": 64,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
    }
  }
  testID="surface-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "rgba(255, 251, 254, 1)",
        "flex": 1,
        "flexDirection": "row",
        "paddingBottom": undefined,
        "paddingHorizontal": 4,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingTop": undefined,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="surface"
  />
</View>
`;
